books-list.html

```diff
@@ -10,7 +10,7 @@
   <body>
     <nav class="navbar">
       <img src="images/w-logo.png" alt="Wolox logo" class="w-logo">
-      <button class="logout">Logout</button>
+      <button class="logout" onclick="window.location='login.html'">Logout</button>
```

Comment:
I think I said it should be a button, so sorry, but it was OK as a link to the `login.html` 🙏